### PR TITLE
Add interactive PPI simulator to the landing page

### LINKED ISSUE #324: Interactive PPI simulator for the landing page
TODO : 
- à mettre à jour depuis les dernières modifications de la landing page
- Investigate clean way to include the dependency on Node in the repository (uv like)
- Update .claude.md with elements about the new make command

## Background

The landing page sells GLIDE's core promise: a small human-labeled sample plus a biased LLM-judge proxy yields a tight, debiased confidence interval. This ticket adds an interactive widget that lets a visitor *feel* that promise by dragging sliders and watching three error bars move: a wide human-only interval, a misleadingly tight but biased judge-only point, and a narrow, unbiased PPI interval.

The page is fully static (GitHub Pages, no backend), so the Python `glide-py` code cannot run server-side. The widget reimplements GLIDE's prediction-powered inference (PPI) math in a small vanilla JavaScript module and is guarded by an integration test checking it agrees with `glide-py` on the quantities it displays. The widget takes five inputs (total dataset size, human-labeled sample size, two accuracies, and the human/judge correlation) and treats the evaluated metric as a binary accuracy, so each variance is fixed by its mean: for a proportion `p` the variance is `p·(1-p)`. In math notation this is written as `Var(Y) = p(1 - p)`, `Var(f) = q(1 - q)`, and `Cov(Y, f) = ρ·√(Var(Y)·Var(f))`, where `p` is the human accuracy, `q` the judge accuracy, and `ρ` their correlation. From these population moments the widget computes each estimate's confidence interval in closed form, mirroring GLIDE's estimators in the large-sample limit.

## Design choices

Everything lives under `docs/landing/`, alongside the existing static page: a new ES module `docs/landing/ppi.js` holds the math, and the existing `docs/landing/index.html` gains a simulator section plus `<script>` tags. The module exposes a single function, `simulate(params)`: the **analytical** closed form that powers the UI. It derives the Bernoulli population moments from the five inputs and plugs them into GLIDE's variance formulas (asymptotic limit), so it is deterministic and updates smoothly as sliders move. It is deliberately the *minimal* equivalent of `glide-py`: it computes only the quantities the widget displays, and ports nothing the display does not show. Its correctness is pinned by testing that displayed perimeter (the PPI and human-only half-widths and the effective sample size) against `glide-py` on a large synthetic dataset.

The chart uses Chart.js with the `chartjs-chart-error-bars` plugin, both loaded from a CDN with pinned versions and SRI hashes, matching the page's existing Bootstrap/Font Awesome pattern. Tests run on Node's built-in runner (`node --test`, zero dependencies); fixtures are generated once by a committed Python script using the real `glide-py` estimators and simulators and checked in as JSON, so CI needs only Node, not a Python/Node bridge.

Deployment: this ticket depends on the Pages allow-list refactor in `tickets/ci-pages-publish-allowlist.md`, which ships first and makes `.github/workflows/pages.yml` publish only an explicit set of staged files. Building on that, this ticket adds a single line — `cp docs/landing/ppi.js _site/` — to the workflow's "Stage site" step so the new module is published, and relies on the allow-list to keep the new `docs/landing/tests/` directory off the live site. The `node --test` run belongs in a separate CI job, gated on `docs/landing/**`.

### `docs/landing/ppi.js` — the analytical `simulate` (the whole module)

**Location:** `docs/landing/ppi.js`

**Purpose:** A closed-form, population-limit reimplementation of GLIDE's PPI math — the minimal surface the widget needs to render its three bars. It derives the Bernoulli population moments from the five slider inputs and plugs them into GLIDE's variance formulas, so the output is deterministic and updates smoothly as sliders move. This is the *only* module the page loads, and the *only* code under test; it deliberately ports nothing from `glide-py` that the display does not show.

```javascript
export const Z_SCORE_95 = 1.959963984540054;

// Fréchet bounds: a covariance between two Bernoulli(p), Bernoulli(q) variables is feasible
// only within [max(0, p + q - 1) - p*q, min(p, q) - p*q]. Clamp so impossible correlations
// (e.g. rho = 1 with p != q) degrade gracefully instead of producing a negative variance.
function clampCovariance(rawCovariance, trueMean, proxyMean) {
  const lowerCovariance = Math.max(0, trueMean + proxyMean - 1) - trueMean * proxyMean;
  const upperCovariance = Math.min(trueMean, proxyMean) - trueMean * proxyMean;
  const clamped = Math.min(Math.max(rawCovariance, lowerCovariance), upperCovariance);
  return clamped;
}

/**
 * Simulate the three estimators (human-only, judge-only, PPI) from population parameters.
 *
 * @param {Object} params
 * @param {number} params.totalSize      Total proxy-labeled dataset size N (e.g. 3300).
 * @param {number} params.humanSize      Human-labeled subset size n, 0 < n < N (e.g. 265).
 * @param {number} params.trueMean       Human accuracy p in (0, 1).
 * @param {number} params.proxyMean      Judge accuracy q in (0, 1).
 * @param {number} params.correlation    Human/judge correlation rho in [-1, 1].
 * @returns {{humanOnly: {mean: number, halfWidth: number}, judgeOnly: {mean: number, halfWidth: number}, ppi: {mean: number, halfWidth: number}, effectiveSampleSize: number, powerMultiplier: number}}
 */
export function simulate({ totalSize, humanSize, trueMean, proxyMean, correlation }) {
  if (!(humanSize > 0 && humanSize < totalSize)) {
    throw new Error(`'humanSize' must satisfy 0 < humanSize < totalSize; got ${humanSize}.`);
  }
  for (const [name, value] of [["trueMean", trueMean], ["proxyMean", proxyMean]]) {
    if (!(value > 0 && value < 1)) {
      throw new Error(`'${name}' must be in (0, 1); got ${value}.`);
    }
  }

  const numberUnlabeled = totalSize - humanSize;
  const trueVariance = trueMean * (1 - trueMean);
  const proxyVariance = proxyMean * (1 - proxyMean);
  const rawCovariance = correlation * Math.sqrt(trueVariance * proxyVariance);
  const labeledCovariance = clampCovariance(rawCovariance, trueMean, proxyMean);

  // Power-tuned lambda (PPI++) with population moments: lambda* = cov / ((1 + n/N) * Var(f)).
  const factor = 1 + humanSize / numberUnlabeled;
  const tuningParameter = labeledCovariance / (factor * proxyVariance);

  // PPI standard error: the asymptotic limit of glide's _compute_std_estimate with population
  // variances. Var(Y - lambda*f) = Var(Y) - 2*lambda*Cov + lambda^2 * Var(f).
  const residualVariance = trueVariance - 2 * tuningParameter * labeledCovariance + tuningParameter ** 2 * proxyVariance;
  const ppiVariance = residualVariance / humanSize + (tuningParameter ** 2 * proxyVariance) / numberUnlabeled;
  const ppiStd = Math.sqrt(ppiVariance);

  // Human-only (classical) and judge-only (naive: treats every proxy label as ground truth).
  const humanStd = Math.sqrt(trueVariance / humanSize);
  const judgeStd = Math.sqrt(proxyVariance / totalSize);

  // Effective sample size, mirroring glide: floor(n * Var_classical / Var_ppi),
  // with Var_classical = Var(Y)/n, hence floor(Var(Y) / Var_ppi).
  const effectiveSampleSize = Math.floor(trueVariance / ppiVariance);

  const result = {
    humanOnly: { mean: trueMean, halfWidth: Z_SCORE_95 * humanStd },
    judgeOnly: { mean: proxyMean, halfWidth: Z_SCORE_95 * judgeStd },
    ppi: { mean: trueMean, halfWidth: Z_SCORE_95 * ppiStd },
    effectiveSampleSize: effectiveSampleSize,
    powerMultiplier: effectiveSampleSize / humanSize,
  };
  return result;
}
```

### `docs/landing/index.html` — simulator section

**Location:** `docs/landing/index.html` (new section in the right-hand `col-md-7` main column, after "Get started", before `</div><!-- /main -->`)

**Purpose:** Five labeled range sliders, a Chart.js bar-with-error-bars chart, and a results readout, wired to `simulate`.

- Add the CDN scripts before `</body>`, version-pinned with SRI hashes (same style as the existing Bootstrap/Font Awesome links):
  - `chart.js` (UMD build).
  - `chartjs-chart-error-bars` (registers the `barWithErrorBars` controller).
  - `<script type="module">` that imports `simulate` from `./ppi.js`, reads the sliders, and renders.
- Sliders (label → `simulate` param), reusing the existing `.hero-card` / `.install-block` styling and `#3a0ca3` / `#4361ee` palette:
  - Total Evaluation Dataset Size (N) → `totalSize` (e.g. 200–10000, step 100).
  - Human-Labeled Sample Size (n) → `humanSize` (e.g. 20–1000, step 5; must stay `< totalSize`).
  - Naive LLM Judge Score → `proxyMean` (0.50–0.99, step 0.01).
  - Observed Human Accuracy → `trueMean` (0.50–0.99, step 0.01).
  - Human/Judge Correlation → `correlation` (0–1, step 0.01) — not in the mockup screenshot but required.
- Chart: one `barWithErrorBars` dataset with three categories — "Human only", "Judge only", "PPI" — each bar value the estimate `mean` and each error bar `[mean - halfWidth, mean + halfWidth]`. Colour PPI in the brand violet to echo the "GLIDE certified" card.
- Readout below the chart (mirroring the mockup's right panel): PPI estimate as `mean ± halfWidth`, the 95% PPI interval, the human-only half-width, the power multiplier (`powerMultiplier` as a percentage), and the **effective sample size**.
- On any slider `input` event, recompute via `simulate` and update the chart and readout. Render once on `DOMContentLoaded`.

### `docs/landing/tests/generate_fixtures.py` — fixture generator

**Location:** `docs/landing/tests/generate_fixtures.py`

**Purpose:** Run the real `glide-py` estimators on large synthetic datasets and write `docs/landing/tests/fixtures.json`, capturing only the displayed perimeter (PPI half-width, human-only half-width, effective sample size). Run by maintainers when the math changes; the JSON is committed so CI needs only Node.

```python
import json
from pathlib import Path

import numpy as np

from glide.estimators import ClassicalMeanEstimator, PPIMeanEstimator
from glide.simulators import generate_binary_dataset

Z_SCORE_95 = 1.959963984540054

# Each case mirrors the five simulator inputs. totalSize is large so glide's finite-sample
# estimate and the analytical population limit in `simulate` agree within the test tolerances.
CASES = [
    {"totalSize": 20000, "humanSize": 4000, "trueMean": 0.77, "proxyMean": 0.76, "correlation": 0.6},
    # Add 1-2 more cases: a different correlation, a more uneven human/total split.
]


def build_fixtures() -> dict:
    simulate_fixtures = []
    for case in CASES:
        y_true, y_proxy = generate_binary_dataset(
            n_total=case["totalSize"],
            true_mean=case["trueMean"],
            proxy_mean=case["proxyMean"],
            correlation=case["correlation"],
            random_seed=0,
        )
        y_true_masked = y_true.copy()
        y_true_masked[case["humanSize"] :] = np.nan
        ppi_result = PPIMeanEstimator().estimate(y_true_masked, y_proxy, power_tuning=True)
        human_result = ClassicalMeanEstimator().estimate(y_true[: case["humanSize"]])
        simulate_fixtures.append(
            {
                "params": case,
                "ppi_half_width": Z_SCORE_95 * ppi_result.std,
                "human_half_width": Z_SCORE_95 * human_result.std,
                "effective_sample_size": ppi_result.effective_sample_size,
            }
        )
    fixtures = {"simulate": simulate_fixtures}
    return fixtures


if __name__ == "__main__":
    output_path = Path(__file__).parent / "fixtures.json"
    fixtures = build_fixtures()
    output_path.write_text(json.dumps(fixtures, indent=2))
```

### `docs/landing/tests/ppi.test.mjs` — integration and unit tests

**Location:** `docs/landing/tests/ppi.test.mjs` (run with `node --test docs/landing/tests/`)

**Purpose:** Pin `simulate`'s displayed perimeter to `glide-py` on a large synthetic dataset (within the population-limit tolerance), and check its exact limiting behaviour at zero correlation.

```javascript
import { test } from "node:test";
import assert from "node:assert/strict";
import { readFileSync } from "node:fs";
import { simulate } from "../ppi.js";

const fixtures = JSON.parse(readFileSync(new URL("./fixtures.json", import.meta.url)));

test("simulate matches glide-py on a large synthetic dataset", () => {
  for (const fixture of fixtures.simulate) {
    const output = simulate(fixture.params);
    const ppiError = Math.abs(output.ppi.halfWidth - fixture.ppi_half_width) / fixture.ppi_half_width;
    assert.ok(ppiError < 0.02);
    const humanError = Math.abs(output.humanOnly.halfWidth - fixture.human_half_width) / fixture.human_half_width;
    assert.ok(humanError < 0.02);
    const essError = Math.abs(output.effectiveSampleSize - fixture.effective_sample_size) / fixture.effective_sample_size;
    assert.ok(essError < 0.05);
  }
});

test("simulate with zero correlation recovers the human-only estimate", () => {
  const output = simulate({ totalSize: 3300, humanSize: 265, trueMean: 0.77, proxyMean: 0.76, correlation: 0 });
  assert.ok(Math.abs(output.ppi.halfWidth - output.humanOnly.halfWidth) < 1e-12);
  assert.equal(output.effectiveSampleSize, 265);
});
```

### `Makefile` — `landing-tests` target

**Location:** `Makefile`

**Purpose:** A dedicated target to run the widget's Node tests, kept separate from `make tests`.

Add:

```makefile
landing-tests:
	node --test docs/landing/tests/
```

Keep it out of the default `make tests` (which stays `unit-tests functional-tests`, pure pytest under `uv`). This mirrors the existing `test-notebooks` target, which is likewise a specialised suite excluded from `make tests`. Two concrete reasons:

- `make tests` runs entirely through `uv run pytest`; the landing tests need Node, which most contributors to a Python library will not have installed. Folding them in would break `make tests` on a stock Python dev setup.
- The widget is a self-contained, zero-dependency mini-project with its own CI job (gated on `docs/landing/**`); a dedicated `make landing-tests` matches that boundary and is what the CI job invokes.

The target runs only the committed tests; it does not regenerate fixtures (that is a manual maintainer step via `generate_fixtures.py`).

### `.github/workflows/landing.yml` — Node test CI

**Location:** `.github/workflows/landing.yml` (new workflow)

**Purpose:** Run `make landing-tests` on pull requests (and pushes to `main`) that touch the landing widget. A dedicated, path-filtered workflow — not a step in `code_quality.yml` — so the Python pipeline stays Node-free and only this job sets up Node. Mirrors the existing `pages.yml`, which is likewise a standalone workflow gated on `docs/landing/**`.

```yaml
name: Landing widget tests

on:
  push:
    branches: [main]
    paths:
      - "docs/landing/**"
  pull_request:
    branches: [main]
    paths:
      - "docs/landing/**"

concurrency:
  group: ${{ github.workflow }}-${{ github.ref }}
  cancel-in-progress: true

permissions:
  contents: read

jobs:
  landing-tests:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v7.0.0
        with:
          persist-credentials: false

      - name: Set up Node
        uses: actions/setup-node@v4
        with:
          node-version: "22"

      - name: Run landing tests
        run: make landing-tests
```

Notes:
- No dependency install step: the widget and its tests are zero-dependency, so a Node runtime is all that is needed (`node --test` is built in on Node 18+; pin a current LTS such as 22).
- Branch-protection caveat: a `paths`-filtered required check never reports on PRs that do not touch `docs/landing/**`, which can leave such PRs waiting on a status that never arrives. If this job is added to required checks, account for that (e.g. do not mark it required, or use a path-aware gating pattern).

## Functional tests

**`simulate` matches `glide-py` on the displayed perimeter**

The only quantities the widget renders — the PPI half-width, the human-only half-width, and the effective sample size — must agree with `glide-py` run on a large synthetic dataset drawn from the same five parameters (via `generate_binary_dataset` with a fixed seed). Because `simulate` is the population/asymptotic limit while `glide-py` runs on a finite sample, the half-widths are checked within a `2%` relative tolerance at `totalSize = 20000` and the effective sample size within `5%`. This is the single integration guarantee, produced by `generate_fixtures.py` and covered by `test("simulate matches glide-py on a large synthetic dataset")` above.

**Zero correlation collapses PPI to human-only**

When `correlation = 0`, the power-tuned lambda is `0`, so the PPI half-width equals the human-only half-width exactly and the effective sample size equals the human sample size. This limiting case is exact (no sampling involved) and is covered by `test("simulate with zero correlation recovers the human-only estimate")` above.

## Corner cases

- `humanSize >= totalSize` or `humanSize <= 0`: `simulate` raises (the slider max for `humanSize` must also be clamped to `totalSize - step` in the UI).
- `trueMean` or `proxyMean` outside `(0, 1)`: `simulate` raises (sliders are bounded to `0.50–0.99`, so this guards programmatic misuse).
- Infeasible correlation (e.g. `correlation = 1` with `trueMean != proxyMean`): `clampCovariance` clamps to the Fréchet bound; no negative variance, no `NaN`.
- `correlation = 0`: the power-tuned lambda is `0`, the PPI half-width equals the human-only half-width, and the effective sample size equals `humanSize`.
- Judge-only bar is intentionally centered at `proxyMean` (not `trueMean`) with a tiny half-width, visualising a confidently-wrong estimate; this contrast is the teaching point and must not be "corrected".

## Acceptance criteria

- [ ] `docs/landing/ppi.js` exports a single function `simulate` as documented; `index.html` loads it as an ES module and renders the simulator section. No `glide-py` machinery beyond what the display uses is ported.
- [ ] `simulate` returns three bars (human-only, judge-only, PPI) with `mean` and `halfWidth`, plus `effectiveSampleSize` and `powerMultiplier`.
- [ ] On the displayed perimeter (PPI half-width, human-only half-width, effective sample size), `simulate` agrees with `glide-py` on a large synthetic dataset within the documented tolerances (`2%` for the half-widths, `5%` for the effective sample size).
- [ ] `simulate` with `correlation = 0` yields a PPI half-width equal to the human-only half-width and `effectiveSampleSize == humanSize`.
- [ ] The five sliders are wired to `simulate`; moving any slider updates the chart (three bars with error bars) and the readout (PPI estimate `± halfWidth`, 95% interval, human-only half-width, power multiplier, effective sample size) with no console errors.
- [ ] A `make landing-tests` target runs `node --test docs/landing/tests/`, kept out of `make tests` (mirroring `test-notebooks`).
- [ ] A new `.github/workflows/landing.yml`, path-filtered on `docs/landing/**`, sets up Node and runs `make landing-tests` on pushes to `main` and pull requests; `code_quality.yml` is left unchanged (no Node added to the Python pipeline).
- [ ] CDN scripts (Chart.js, error-bars plugin) are version-pinned with SRI hashes, matching the existing CDN-link style in `index.html`.
- [ ] The "Stage site" step in `pages.yml` gains `cp docs/landing/ppi.js _site/`, so the module is published; test files under `docs/landing/tests/` remain unpublished via the allow-list from `tickets/ci-pages-publish-allowlist.md` (a dependency of this ticket).
```

### PR DESCRIPTION
**Pull request by imerad**
### Description

Adds an interactive widget to the landing page that lets visitors drag sliders (dataset size, human-labeled sample size, two accuracies, human/judge correlation) and watch three live confidence intervals update: a wide human-only interval, a misleadingly tight but biased judge-only point, and a narrow, unbiased PPI interval. The PPI math is reimplemented in a small vanilla JavaScript module (`docs/landing/ppi.js`), validated by a Node test suite (`docs/landing/tests/`) against fixtures generated from `glide-py`, with CI wiring to run and publish it.

Handles this [ticket](https://github.com/orgs/EmertonData/projects/26/views/2?pane=issue&itemId=188675850&issue=EmertonData%7Cglide%7C324) Closes #324

Run the commands below to test locally : 
```
# From the repo root, mirror the same staging steps pages.yml uses in CI
mkdir -p _site/assets
cp docs/landing/index.html _site/
cp docs/landing/ppi.js _site/
cp docs/assets/*.png docs/assets/*.jpg _site/assets/

# Serve it (either works)
python3 -m http.server 8000 --directory _site
# or: npx serve _site
```
Then open http://localhost:8000

### Checklist

Quality gates that must be satisfied before requesting a review:

- [x] I have read `CONTRIBUTING.md`
- [ ] `make lint` passes
- [ ] `make type-check` passes
- [ ] `make tests` passes
- [ ] `make coverage` reports 100% coverage
- [ ] `make test-notebooks` passes
- [ ] New public API has numpy-style docstrings
- [ ] New public API is inserted in the API reference section of the documentation
- [ ] Docs build without warnings (`make doc`)
- [ ] `CHANGELOG.md` updated if the change is user-facing

### LLM usage

Disclose if an LLM was used in writing this PR:
- [ ] No LLM used
- [x] I used an LLM and I went through and validated all the code myself

### CODE REVIEW

**Comment by imerad** [comment]
## Code review

Verified the ported PPI math in `docs/landing/ppi.js` line-by-line against `glide/estimators/ppi_core.py` and `ppi.py` (tuning parameter, rectified mean/variance, effective sample size) and against the Fréchet–Hoeffding covariance bounds used in `getCorrelationBounds`. No correctness bugs found in the math itself. A few smaller issues below (including minor/nitpick ones, as requested).

### Consistency / validation
- **Missing input validation on `correlation` in `simulate()` and on `trueMean`/`proxyMean` in `getCorrelationBounds()`** (`docs/landing/ppi.js`). `simulate()` validates `humanSize`, `trueMean`, `proxyMean` at the top but not `correlation` (docstring claims `rho ∈ [-1, 1]`), and `getCorrelationBounds()` validates nothing despite relying on the same `(0,1)` precondition as `simulate()`. Low risk today since the UI clamps slider ranges before calling either function, but worth mirroring `simulate()`'s validation pattern for consistency if this module is reused elsewhere.
- **`docs/landing/tests/generate_fixtures.py:29`** — `def build_fixtures() -> dict:` uses the bare builtin `dict` instead of `Dict[str, ...]` from `typing`. The repo's type-annotation convention (no PEP 585 builtin generics) applies repo-wide, not just under `glide/`. Won't be caught by `ruff` (only `E`/`I` rules are selected here), so it's a manual-review-only nit.

### CI / drift risk
- Nothing in CI regenerates and diffs `fixtures.json` against `generate_fixtures.py`. If `ppi_core.py`'s formulas ever change, `fixtures.json` could silently drift out of sync with the Python source of truth, and the Node tests would keep "passing" against stale fixtures. Consider a comment in `generate_fixtures.py` (or a lightweight CI check) reminding maintainers to regenerate fixtures whenever `glide.estimators.ppi_core` / `PPIMeanEstimator` changes.
- `docs/landing/tests/fixtures.json` has no trailing newline (`\ No newline at end of file` in the diff) — minor style nit.

### Test coverage
- The math in `ppi.js` (`simulate`, `getCorrelationBounds`, clamping) is well covered by the Node suite, including parity against real `glide-py` fixtures, the zero-correlation edge case, both validation error paths, and infeasible-correlation clamping.
- The DOM-wiring/rendering code inlined in `index.html` (`render`, `updateChart`, `clampHumanSize`, `clampCorrelation`, `updateReadout`, `trueMeanLinePlugin`) has no automated test coverage. Reasonable for a landing page given the low blast radius and that the underlying math is tested, but worth a manual smoke test (drag each slider to its min/max, including the correlation slider when `trueMean` == `proxyMean`) since it can't run through `make coverage`.

### Checklist
The PR body currently has `make lint`, `make type-check`, `make tests`, `make coverage`, and `make test-notebooks` all unchecked — worth confirming these pass before merge, particularly for the new `generate_fixtures.py` Python file.

### Nice to have (not blocking)
- `.github/workflows/landing.yml` / `pages.yml` correctly copy `ppi.js` alongside `index.html` for deployment, and the Chart.js / `chartjs-chart-error-bars` CDN includes use SRI `integrity` hashes — good practice, no action needed, just noting it was checked.

File: .github/workflows/landing.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+name: Landing widget tests
```
Comment (gmartinon-ed) [inline .github/workflows/landing.yml:1]: Génial ce simulateur! Quelques retours visuels:

Peut-on renommer les sliders:
- sample size
- Number of ground truths
- Ground truth estimate
- Proxy estimate
- Ground truth / proxy correlation

( à propager dans la note)

Peut-on augementer la font size du graphe pour le rendre plus lisible ? et aussi du texte autour des sliders ?
Les trois barres, on sent qu'il y a des couleurs différents, mais c'est pas clair. Prendre dans cet ordre les couleurs du thème Glide : dark blue, light blue, yellow

On voit à peine les point estimates, peut-on grossir les points?

Peut-on épaissir les traits du forest plot ? 

je vois que quand on passe la souris, il y a une text box. Elle est un peu illisible car il y a trop de chiffres après la virgule (trois suffisent). De plus la couleur du carré en haut à gauche de la text box n'est pas la même que la couleur du tracé, il faut homogénéiser. Sur le texte, je verrais un truc du style:
Point estimate: 0.XXX
Lower bound (95% confidence): 0.YYY
Upper bound (95% confidenceà: 0.ZZZ

Sur l'encadré noir en bas, je pense que les deux dernières lignes se suffisent à elle-même, le reste est redondant avec le graphe. Je mettrais un truc du genre:
Effective sample size: XXX
GLIDE's estimate is as precise as if you had YYY ground truth annotations, instead of the provided ZZZ

Enfin, j'inverserai la section "Try it yourself" et "get started". Les gens vont d'abord jouer avec le simulateur et ensuite installer.

Aussi: sur les range des sliders:
- size : Peut-on démarrer à 50, aller de 50 en 50, jusqu'à 2000 ?
- groud truth: on ne peut pas aller jusqu'à 100% du dataste size, normal ?

Sur la corrélation, peut-on garder les extrémités fixe (la gauche représente 0, la droite 1), mais réduire horizontalement le span possible en fonction des averages ?

File: .claude/CLAUDE.md [hunk 2]
```diff
@@ -22,6 +22,8 @@ make coverage      # Coverage report on unit tests only (100% required; function
 make pre-commit    # Run pre-commit hooks via prek (not the standard pre-commit CLI)
 make test-notebooks # Test all Jupyter notebooks
 make doc            # Build and serve documentation locally with MkDocs
+make landing-tests  # Run Node tests for the landing-page PPI widget (requires Node; see .nvmrc)
+make check-landing-fixtures # Regenerate docs/landing/tests/fixtures.json from glide-py and fail if it drifted from the committed version
```
Comment (gmartinon-ed) [inline .claude/CLAUDE.md:26]: Quesaco ?

File: .nvmrc [hunk 3]
```diff
@@ -0,0 +1 @@
+22
```
Comment (gmartinon-ed) [inline .nvmrc:1]: Quesaco ?

**Review by gmartinon-ed — CHANGES_REQUESTED** [review]

File: .github/workflows/landing.yml (hunk 1 above)
Comment (imerad) [reply .github/workflows/landing.yml:1]: En principe tous ces feedbacks ont été appliqués. Concernant le size jusqu'à 2000, noter que le gain de PPI semble minime à cette valeur. La taille du ground truth était limitée exprès, mais maintenant elle peut aller (presque) jusqu'au size total (Une valeur égale provoquerait une erreur dans le javascript).

File: .claude/CLAUDE.md (hunk 2 above)
Comment (imerad) [reply .claude/CLAUDE.md:26]: La première commande teste le code javascript qui fait la simulation (le ppi.js). La deuxième teste que le script python génère bien les mêmes fixtures traquées dans le repo.

File: .nvmrc (hunk 3 above)
Comment (imerad) [reply .nvmrc:1]: Un fichier qui indique la version de Node utilisée. Apparemment nécessaire pour le développement du simulateur.

File: docs/landing/index.html [hunk 4]
```diff
@@ -247,16 +247,28 @@
       display: flex;
```
Comment (gmartinon-ed) [inline docs/landing/index.html:247]: Le jaune de la barre glide est trop terne, prendre le jaune le plus clair de la palette du logo (ça devrait être le même jaune que le carré qui s'affiche quand on passe la souris)

Peut-on grossir les points estimate encore ? Ils sont à peine plus gros que l'épaisseur des traits. Et ils ont des couleurs de contour bleues il me semble, peut-on ne pas mettre de contour, ou alors blanc ?

Dans les encadrés qui s'affichent quand on passe la souris, il y a deux fois le même texte.
En fait, c'est un peu dur à lire ces upper/lower bounds. Peut-on mettre à la place
0.XXX +/- 0.YYY (95% confidence)
?

Sur les sliders, les contraintes conditionnels (n_true < n_total, correlation < max_correlation) sont rendues différemment. Peut-homogénéiser et aligner sur les coorélations ?

Le label PPI: PPI n'est pas défini, donc mettre "GLIDE"

Enfin, je pense qu'il faut une petite note qui dise "error bar can even be reduced further if you consider stratification or multiple proxies"

**Review by gmartinon-ed — CHANGES_REQUESTED** [review]

File: docs/landing/index.html (hunk 4 above)
Comment (imerad) [reply docs/landing/index.html:247]: Remarques prises en compte

**Review by gmartinon-ed — APPROVED** [review]
